devel/cmake-devel/Portfile

```diff
@@ -361,11 +360,14 @@ subport ${subport_docs} {
 
 if {${subport} eq ${name}} {
     notes "\
-        The CMake Gui and Docs are now provided as subports '${subport_gui}' and '${subport_docs}', respectively.
+        The CMake GUI and Docs are now provided as subports '${subport_gui}' and '${subport_docs}', respectively.
     "
 
     # Ignore RC versions
     gitlab.livecheck.regex  {([0-9.]+)}
 } else {
     livecheck.type          none
 }
+
+test.run            yes
+test.target         test
```

Comment:
You'll only want to enable tests for the main port.

Reply:
> I am not experienced with subports. Do I simply wrap this in a `if {${subport} eq ${name}}` conditional, separately from the livecheck above?

Yes indeed!

Reply:
OK, fixed (though I added it together with the livecheck block).